examples/react/basic/package.json

```diff
@@ -16,7 +16,7 @@
     "@bufbuild/protobuf": "^1.3.3",
     "@bufbuild/protoc-gen-es": "^1.3.3",
     "@connectrpc/connect": "^1.1.2",
-    "@connectrpc/connect-query": "workspace:*",
+    "@connectrpc/connect-query": "0.5.1",
```

Comment:
This should be 0.6.0, I think. 

Can we simply use `*` as the version constraint in the example? Definitely not ideal, but perhaps better than having to manually update the field for a a version bump.

Reply:
Yeah that's kind the sideeffect of having an example that points to a real release. Without the actual release being done, we can't point to most recent here.

Reply:
Returning to workspace for now. Will need to think on a better way to keep these in sync without using a workspace reference.

Reply:
Ah, so pnpm only links the `workspace:*` reference? That's a bummer.

Reply:
Sorta bummer but also nice and predictable, it prevents accidentally using one over the other.